.gitignore

```diff
@@ -1,3 +1,6 @@
+# OS's
+.DS_Store
```

Comment:
these are all over the place on macOS, hope you don't mind

Reply:
I personally don't mind.
Though in the best practice, you could add those to your personal gitignore (global git config).